### PR TITLE
Install: bring build doc up to date for style

### PR DESCRIPTION
**Pull request by esmerel**
Brings the page up to general style guidelines, but I didn't add most of the things to the spelling dictionary.

### CODE REVIEW

File: installation/sources/build-and-install.md [hunk 1]
```diff
@@ -1,223 +1,243 @@
...
-[Fluent Bit](http://fluentbit.io) uses [CMake](http://cmake.org) as its build system. The suggested procedure to prepare the build system consists of the following steps:
+[Fluent Bit](http://fluentbit.io) uses [CMake](http://cmake.org) as its build system.
 
 ## Requirements
 
-- CMake >= 3.12
+- CMake 3.12 or greater. You might need to use `cmake3` instead of `cmake`.
 - Flex
-- Bison >= 3
+- Bison 3 or greater
```
Comment (lockewritesdocs) [inline installation/sources/build-and-install.md:9]: 🦬 🦬 🦬

File: installation/sources/build-and-install.md [hunk 2]
```diff
@@ -1,223 +1,243 @@
...
--- Looking for accept4
--- Looking for accept4 - not found
--- Configuring done
--- Generating done
--- Build files have been written to: /home/edsiper/coding/fluent-bit/build
-```
-
-Now you are ready to start the compilation process through the simple _make_ command:
+If you already know how CMake works you can skip this part and look at the build
+options available.
```
Comment (lockewritesdocs) [inline installation/sources/build-and-install.md:16]:
```suggestion
If you already know how CMake works, you can skip this section and review the
available [build options](#general-options).
```

File: installation/sources/build-and-install.md [hunk 3]
```diff
@@ -1,223 +1,243 @@
...
+  ```
+
+1. Let [CMake](http://cmake.org) configure the project specifying where the root
+   path is located:
+
+  ```bash
+  cmake ../
+  ```
+
+  This should display a series of results like:
```
Comment (lockewritesdocs) [inline installation/sources/build-and-install.md:34]:
```suggestion
  This command displays a series of results similar to:
```

File: installation/sources/build-and-install.md [hunk 4]
```diff
@@ -1,223 +1,243 @@
...
+  -- Build files have been written to: /home/edsiper/coding/fluent-bit/build
+  ```
+
+1. Start the compilation process using the `make` command:
+
+  ```bash
+  make
+  ```
+
+  This should display results like:
```
Comment (lockewritesdocs) [inline installation/sources/build-and-install.md:60]:
```suggestion
  This command displays results similar to:
```

File: installation/sources/build-and-install.md [hunk 5]
```diff
@@ -1,223 +1,243 @@
...
-[ 69%] Building C object src/CMakeFiles/fluent-bit-static.dir/flb_input.c.o
-[ 71%] Building C object src/CMakeFiles/fluent-bit-static.dir/flb_output.c.o
-...
-Linking C executable ../bin/fluent-bit
-[100%] Built target fluent-bit-bin
+make install
 ```
 
-to continue installing the binary on the system just do:
+You might need root privileges. You can try to prefix the command with `sudo`.
```
Comment (lockewritesdocs) [inline installation/sources/build-and-install.md:87]:
```suggestion
If the command indicates insufficient permissions, prefix the command with `sudo`.
```

File: installation/sources/build-and-install.md (hunk 2 above)
Comment (lockewritesdocs) [reply installation/sources/build-and-install.md:16]: Suggest adding a nice little cross link.

File: installation/sources/build-and-install.md [hunk 6]
```diff
@@ -1,223 +1,243 @@
...
-| FLB\_MSGPACK\_TO\_JSON\_INIT\_BUFFER\_SIZE | Determine initial buffer size for msgpack to json conversion in terms of memory used by payload. | 2.0 |
-| FLB\_MSGPACK\_TO\_JSON\_REALLOC\_BUFFER\_SIZE | Determine percentage of reallocation size when msgpack to json conversion buffer runs out of memory. | 0.1 |
+| `FLB_MSGPACK_TO_JSON_INIT_BUFFER_SIZE` | Determine initial buffer size for `msgpack` to `json` conversion in terms of memory used by payload. | `2.0` |
+| `FLB_MSGPACK_TO_JSON_REALLOC_BUFFER_SIZE` | Determine percentage of reallocation size when `msgpack` to `json` conversion buffer runs out of memory. | `0.1` |
 
-### Input Plugins
+### Input plugins
 
-The _input plugins_ provides certain features to gather information from a specific source type which can be a network interface, some built-in metric or through a specific input device, the following input plugins are available:
+Input plugin gather information from a specific source type like network interfaces,
```
Comment (lockewritesdocs) [inline installation/sources/build-and-install.md:139]:
```suggestion
Input plugins gather information from a specific source type like network interfaces,
```

**Review by lockewritesdocs — APPROVED** [review]
Suggested a few changes, but approving to keep the wheels of industry turning 🌐

File: installation/sources/build-and-install.md (hunk 2 above)
Comment (esmerel) [reply installation/sources/build-and-install.md:16]: I'm going to worry about crosslinks in a later iteration :D

**Pull request merged**